singularity-bootstrap-prefix.def

```diff
@@ -1,16 +1,35 @@
 Bootstrap: docker
-From: centos:8.2.2004
+From: awesomebytes/gentoo_prefix_ci_prepare
+Stage: spython-base
+
+# This container installs Gentoo Prefix noninteractively in 
 
 %post
-    dnf install -y gcc gcc-c++ make diffutils
-    chmod 755 /usr/local/bin/bootstrap-prefix.sh
+# Prepared Ubuntu 16.04
 
-%environment
-    export LC_ALL=C
-    export PATH=/usr/local/bin:$PATH
+# Let's get some specs of the machine that is running this job
+cat /proc/cpuinfo && cat /proc/meminfo && df -h
+
+mkdir /scratch
 
-%files
-    bootstrap-prefix.sh /usr/local/bin
+cd /usr/local/bin
+# Get EESSI or Official bootstrap script
+# wget https://gitweb.gentoo.org/repo/proj/prefix.git/plain/scripts/bootstrap-prefix.sh
+wget https://raw.githubusercontent.com/EESSI/compatibility-layer/master/bootstrap-prefix.sh
+chmod +x bootstrap-prefix.sh
+
+%environment
+#   export EPREFIX=/scratch/gentoo
+   export LC_ALL=C
+   export PATH=/usr/local/bin:$PATH
 
 %runscript
```

Comment:
Maybe we should check here if ${EPREFIX} is defined?

Reply:
This was a bad commit and should be reverted. EPREFIX should not be defined in the container.
The container takes the same arguments as the bootstrap prefix script:
`singularity-bootstrap-prefix.sif $EPREFIX noninteractive`